bundles/io/org.openhab.io.caldav/src/main/resources/readme.txt

```diff
@@ -0,0 +1 @@
+Bundle resources go in here!
```

Comment:
Lots of bundles have it so I assumed there is a reason, plus it's a trivial size. It would be good to know but probably OK to leave it (IMHO).